free-2.md

```diff
@@ -58,12 +72,15 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+A significant amount of time would have to be spent researching people's needs and figuring out how to turn those into coherent, fluent commands. Inspiration could be taken from python's existing library to see what sort of data manipulation is possible and how they are carried out. 
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
+The idea involves several distinct steps, all of which can be very challenging. I have no idea how to parse through a PDF, let alone isolate the tables, although I know it is possible because it's been done already - though quite sub-optimally. The hardest part in my opinion would be recreating those equations and dependencies between cells, and doing it in a way that makes it easier than the way it's currently done in Excel. 
 
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+If I do choose to pursue this option, I have a great group of people with industry experience (CMC alumni, among others) to interview and get to the root of their needs. It would be a very real solution to a commonly experienced problem. This may not be a good idea because I am unsure of how well one could even simplify writing Excel macros and whether this idea is truly feasible. The people I have spoken with are not programmers so it is difficult for them to imagine what is and isn't feasible with a DSL. 
```

Comment:
Having done a lot of work with Excel spreadsheets and working automation into them, I'd recommend avoiding this project (at least for this class). There is a ton of annoying systems-level work that needs to be done and I think it would hide away the language-design part. Additionally, you've been fairly vague about what the language would actually look like, and I think you need to have a more concrete idea there first.